pkgs/by-name/sd/SDL2/package.nix

```diff
@@ -166,7 +166,7 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optional stdenv.hostPlatform.isWindows "--disable-video-opengles"
     ++ lib.optional stdenv.hostPlatform.isDarwin "--disable-sdltest";
 
-  dontDisableStatic = if withStatic then 1 else 0;
+  dontDisableStatic = withStatic;
 
```

Comment:
This seems to be unused now.

```suggestion
```

Reply:
This parameter defines which files will be removed  in `postInstall`.